template/app/controllers/web/my-account.js

```diff
@@ -3,9 +3,12 @@ const _ = require('lodash');
 const cryptoRandomString = require('crypto-random-string');
 const humanize = require('humanize-string');
 const isSANB = require('is-string-and-not-blank');
-const moment = require('moment');
+const dayjs = require('dayjs');
 const { isEmail } = require('validator');
 
+dayjs.extend(require('dayjs/plugin/duration'));
+dayjs.extend(require('dayjs/plugin/relativeTime'));
```

Comment:
You could just do `const dayjs = require('dayjs-with-plugins')` as an alternative, this works too though